src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostInitiateByIdControllerTest.java

```diff
@@ -137,8 +145,130 @@ void should_return_403_with_application_problem_response_when_client_is_not_allo
     }
 
```

Comment:
We don't appear to have a specific test for the AC in the ticket. i.e. when the 2nd thread has a LockTimeOutException and then throws a 503